examples/chef/linux/include/CHIPProjectAppConfig.h

```diff
@@ -32,3 +32,6 @@
 
 // Allows app options (ports) to be configured on launch of app
 #define CHIP_DEVICE_ENABLE_PORT_PARAMS 1
+
+// Enable subscriptions synchronization
+#define CHIP_CONFIG_SYNCHRONOUS_REPORTS_ENABLED 1
```

Comment:
SHould this only be enabled for LIT ICD?

Reply:
I think this is only used in examples/platform/silabs/MatterConfig.cpp . I'll remove this .

Reply:
This is a seperate feature on when we send out subscription report - it could be valuable to use the Synchronised report scheduler since it is in theory a better logic for ICDs

Reply:
Since this feature only works in silabs, it is not required in Linux app, right?

Reply:
No it is not required - It was more a suggestion that this would be a good feature to use while testing ICDs. I doesn't seem like the linux app support its. 

Can be a follow up change or the comment can be ignored.